server/test/services/applications/PdfExporterServiceTest.java

```diff
@@ -71,4 +72,19 @@ public void generateApplicationPdf() throws IOException {
       assertThat(linesFromPDF.get(lineNum)).isEqualTo(linesFromStaticString.get(lineNum));
     }
   }
+
+  @Test
+  public void generateProgramPreviewPdf() throws IOException {
+    PdfExporterService service = instanceOf(PdfExporterService.class);
+
+    PdfExporter.InMemoryPdf result =
+        service.generateProgramPreviewPdf(
+            fakeProgram.getProgramDefinition(), getFakeQuestionDefinitions());
+
+    List<String> linesFromPdf = getPdfLines(result);
+    assertThat(linesFromPdf.get(0))
```

Comment:
```suggestion
    assertThat(linesFromPdf).isNotEmpty();
    assertThat(linesFromPdf.get(0))
```

suggestion: Assert the list is not empty before calling for the first element

Reply:
Updated.